uiowa/uiowa
# Admissions - AoS Tweaks

Resolves #3046 
Resolves #3047 

## To Test

- pull and ds admissions
- goto http://admissions.local.drupal.uiowa.edu/admin/config/search/path/update_bulk select "content" and "update the URL alias for paths having an old URL alias" all existing AoS content should now start with /academics/...
- edit an existing AoS like http://admissions.local.drupal.uiowa.edu/node/761/edit
- notice the old separate label and entity reference fields for academic requirement and application have been replaced with link fields. the title is required if a link (internal or external) is provided.
- upon save you should see button/link type renders similar to the mockups.
